
   
 |PyPI|  |Docs| |PyPIDownloads|


.. |PyPI| image:: https://img.shields.io/pypi/v/epicarousel.svg
   :target: https://pypi.org/project/epicarousel/
.. |Docs| image:: https://readthedocs.org/projects/epicarousel/badge/?version=latest
   :target: https://epicarousel.readthedocs.io
.. |PyPIDownloads| image:: https://pepy.tech/badge/epicarousel
   :target: https://pepy.tech/project/epicarousel
   

EpiCarousel: memory- and time-efficient identification of metacells for atlas-level single-cell chromatin accessibility data
----------------




.. toctree::
   :maxdepth: 2
   :hidden:

   installation
   usage
   api/index
   release/index


**Recent technical advancements in single-cell chromatin accessibility sequencing (scCAS) have brought new insights to the characterization of epigenetic heterogeneity. As single-cell genomics experiments scale up to hundreds of thousands of cells, the demand for computational resources of downstream analysis grows intractably large and exceeds the capabilities of most researchers. Here, we propose EpiCarousel, a tailored Python package for memory- and time-efficient identification of metacells, i.e., the mergence of homogenous cells, in large-scale scCAS data.**


News
----

.. include:: news.rst
   :start-line: 2
   :end-line: 22  